cattqt.py

```diff
@@ -0,0 +1,1227 @@
+# Copyright 2020 - Scott Moreau
```

Comment:
Did you mean to copy this file? Seems like a mistake.

Reply:
I opened a new pull request with the same name that should fix me accidentally copying the whole file. I'll close this one.